pkgs/development/tools/semgrep/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, fetchzip, stdenv }:
+stdenv.mkDerivation rec {
+  pname = "semgrep";
+  version = "0.69.1";
+
+  src = fetchzip {
+    url = "https://github.com/returntocorp/semgrep/releases/download/v${version}/semgrep-v${version}-ubuntu-16.04.tgz";
+    sha256 = "sha256-TTQxChQ/daBCigQkWavCVrZ7BCeZArUH9Tenp7hgQaM=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
```

Comment:
We need no patching at all?

Reply:
They distribute static binaries.

It runs correctly when doing `nix run .#semgrep -- --help`